get-latest-tgf.ps1

```diff
@@ -0,0 +1,40 @@
+$ErrorActionPreference = "Stop" #Make all errors terminating
+
+try {
+    $TGF_PATH = (Get-Command tgf.exe).Path
+    Write-Host tgf path: $TGF_PATH
+    $LOCAL_VERSION = (tgf.exe --current-version) | Out-String
```

Comment:
Why piping in Out-String?

Reply:
Thought it would be necessary. I'll fix it

Reply:
The Out-String is only useful if you want to control the output of the command (change the width ou remove newlines).